src/app/components/code-block/index.html

```diff
@@ -0,0 +1,54 @@
+<sky-demo-page
+  pageTitle="Code block"
+  skyuxModule="SkyCodeBlockModule"
+  packageName="lib-code-block"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The code block component formats a block of text as a code snippet with syntax highlighting. The code block also has a header that displays a copy button and, if defined, the language type. To create a code snippet, wrap <stache-code>sky-code-block</stache-code> tags around the code block.
+    </p>
+    <p>
+      If a code block has text within <stache-code>&lt;</stache-code> and <stache-code>&gt;</stache-code> characters, then the component automatically appends a closing tag unless the code snippet is valid HTML. To prevent this, you must use "&lt;" to escape <stache-code>&lt;</stache-code> and "&gt;" to escape <stache-code>&gt;</stache-code>. For example, you must escape the <stache-code>&lt;</stache-code> and <stache-code>&gt;</stache-code> characters in the code snippet <stache-code>public Task&lt;Generic> MethodName()</stache-code> or else the component automatically appends <stache-code>&lt;/Generic></stache-code> to the end: <stache-code>public Task&lt;Generic> MethodName()&lt;/Generic></stache-code>.
+    </p>
+    <p>
+    Using the list below, see if your code snippet language is supported. The languages supported appear in pairs. The first part lists the language and the second part displays the text you use for the <stache-code>languageType</stache-code> property. For example, if your code block is a JavaScript code snippet, use <stache-code>languageType="javascript"</stache-code>.
```

Comment:
I wonder if we can do without this paragraph. I'd delete it. If you think we need it, we could move it to a section below as a lead-in to the table of language types and tighten it up a bit. ... As for the section with the language types, I will look into what we need to do to set that up.

Reply:
For the language types section, I'm thinking something along the lines of the https://developer.blackbaud.com/skyux/components/status-indicator#status-indicator-classes.

Reply:
Looks like this was addressed, so I'm resolving the comment.